libs/application/templates/health-insurance/src/healthInsuranceUtils.ts

```diff
@@ -42,9 +39,9 @@ export const hasIcelandicAddress = (externalData: ExternalData) => {
 export const shouldShowModal = (externalData: ExternalData) => {
   return (
     hasHealthInsurance(externalData) ||
-    hasActiveApplication(externalData) ||
     hasOldPendingApplications(externalData) ||
```

Comment:
I think we should rename all references to `oldPendingApplications` to `pendingApplications`. For example the provider and id on the healthInsuranceForm

Reply:
Okay, I will change this to be more descriptive to what this function checks now.